apps/site/assets/ts/tnm/components/TransitNearMe.tsx

```diff
@@ -101,11 +95,26 @@ const TransitNearMe = ({
         mapRef.current!.scrollIntoView();
       }
     },
-    [mapData.markers.length]
+    [mapData.markers.length, state.stopsWithRoutes.length]
+  );
+  const selectedStop = getSelectedStop(
+    state.stopsWithRoutes,
+    state.selectedStopId
+  );
+
+  // call once on initial load
+  useEffect(
+    () => {
+      fetchRealtimeSchedules(stopsWithDistances.stops, dispatch);
```

Comment:
So I suppose a trade-off of doing this all clientside is that SSR rendering is moot for the cards. The user experience of the sometimes long page load is worse though.